server/repository/src/migrations/v2_01_00/property.rs

```diff
@@ -0,0 +1,33 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    const PROPERTY_VALUE_TYPE: &str = if cfg!(feature = "postgres") {
+        "property_value_type" // This is created as part of the asset_catalogue_property migration
+    } else {
+        "TEXT"
+    };
+
+    sql!(
+        connection,
+        r#"
+            CREATE TABLE property (
```

Comment:
Do you think we should refactor asset_property to use this too?

Reply:
Probably... was already feeling like a big PR though, and asset properties have just been refactored 😭 

Maybe something for refactor sprint?